services/database.service.ts

```diff
@@ -0,0 +1,92 @@
+require("dotenv").config()
+import { MongoClient, ServerApiVersion } from "mongodb";
+import { Task } from "../models/task.model";
+
+
+const uri: string = `mongodb+srv://${process.env.DB_USER_NAME}:${process.env.DB_USER_PASSWORD}@${process.env.DATABASE_CLUSTER}/?retryWrites=true&w=majority&appName=${process.env.APP_NAME}`;
+const client: MongoClient = new MongoClient(uri, {
```

Comment:
Optional: Refactor the DB connection to use mongoose library, will be much simpler you wont have to open and close connections for each method either:
https://mongoosejs.com/docs/connections.html

I like this implementation aswell though, so if you want to keep it you can